.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
 env:
 - TEST_DIR=.
 - TEST_DIR=scripts/babel-relay-plugin
-script: cd $TEST_DIR && npm install && npm test
+script: cd $TEST_DIR && npm install && npm run lint && npm test
```

Comment:
`npm run lint` will look for a `lint` script in the `$TEST_DIR/package.json`. When `$TEST_DIR` is scripts/babel-relay-plugin` this will fail because there is not `lint` script there. That's *exactly* what's happening: https://travis-ci.org/facebook/relay/jobs/78187439.

I think even doing what you have there will be run from `$TEST_DIR` (should work if you `pushd $TEST_DIR && commands && popd` but then you'll be running top level lint twice)

The other piece of this is that we might want lint to be an allowable failure, which isn't possible like this. In React we put it as a separate test so could (if we needed, which we did for a while) to make it so that lint could fail but jest couldn't. In Relay's case, if there are other things we need to sync out, it might make sense to have the same capability.